form.php

```diff
@@ -0,0 +1,51 @@
+<?php
+?>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Painting Collection</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="css/normalise.css" type="text/css" rel="stylesheet">
+        <link href="css/style.css" type="text/css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,400;0,600;0,700;1,400&display=swap" rel="stylesheet">
+    </head>
+    <body class="form-body">
+        <div class="form-title">
+            <h1>Add Painting To The Collection</h1>
```

Comment:
This page is not part of story 1, please remove this file and add to story2 branch.